doc/UserGuide.md

```diff
@@ -62,9 +62,9 @@ and persons matching at least one keyword will be returned (i.e. `OR` search).
 
 Examples: 
 * `find John`<br>
-  Returns `John Doe` but not `john`
+  Returns both `John Doe` and `john`
 * `find Betsy Tim John`<br>
-  Returns Any person having names `Betsy`, `Tim`, or `John`
+  Returns Any person having names `Betsy`, `Tim`, or `John` regardless of the cases of the letters
```

Comment:
Good. There is a description of the command above that you should update as well.